specification/network/resource-manager/Microsoft.Network/stable/2018-12-01/examples/ExpressRouteCircuitConnectionList.json

```diff
@@ -1,6 +1,5 @@
 {
     "parameters": {
-      "connectionName": "circuitConnectionUSAUS",
```

Comment:
can you update example when there are more than one connection

Reply:
The example is showing two connections.